.github/workflows/prof_controller_kind_test.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build & Apply Profile Controller manifests in KinD
+on:
+  pull_request:
+    paths:
+      - components/profile-controller/config/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install KinD
+      run: |
+        chmod +x components/testing/gh-actions/install_kind.sh
```

Comment:
let's make these script files executables and commit them to git. Git can track the file permissions, so we don't have to `chmod +x` on each GH Action

Reply:
I made all the scripts executable.